packages/client/ui/react-ui/src/hooks/useCrossmint.tsx

```diff
@@ -18,17 +16,14 @@ export function CrossmintProvider({
     const [version, setVersion] = useState(0);
 
     const crossmintRef = useRef<Crossmint>(
-        new Proxy<Crossmint>(
-            createCrossmint({ ...createCrossmintParams, jwt: createCrossmintParams.jwt ?? getCachedJwt() }),
-            {
-                set(target, prop, value) {
-                    if (prop === "jwt" && target.jwt !== value) {
-                        setVersion((v) => v + 1);
-                    }
-                    return Reflect.set(target, prop, value);
-                },
-            }
-        )
+        new Proxy<Crossmint>(createCrossmint(createCrossmintParams), {
+            set(target, prop, value) {
+                if (prop === "jwt" && target.jwt !== value) {
+                    setVersion((v) => v + 1);
+                }
+                return Reflect.set(target, prop, value);
+            },
+        })
```

Comment:
@jmderby Leaving everything as is here, since the unit tests defines a requirement that we can't hit w/o. The updating jwt from SDK piece.